src/Symfony/Bundle/FrameworkBundle/Templating/Helper/FormHelper.php

```diff
@@ -169,6 +169,18 @@ public function label(FormView $view, $label = null, array $variables = array())
         return $this->renderer->searchAndRenderBlock($view, 'label', $variables);
     }
 
+    /**
+     * Renders the help of the given view.
+     *
+     * @param FormView $view The parent view
+     *
+     * @return string The HTML markup
+     */
+    public function help(FormView $view)
```

Comment:
we can use a return type hint here

Reply:
Sorry, I don't understand what you mean... :-(

Reply:
The method definition could look like this:

```php
public function help(FormView $view): string
```

Reply:
Mmm, but it's used anywhere in the file, no ?

Reply:
That's because the existing code was created when we still supported PHP 5. For new features we can rely on PHP 7 features.

Reply:
Oki :-) I do it :-) (I learn something new, thanks)